spec/stat_tracker_spec.rb

```diff
@@ -110,6 +110,11 @@
 
   #League Stat
   describe '#best_offense' do
+    it 'uses the #average_goals method' do
```

Comment:
Excellent helper method that we could use as an example for new methods going forward